libs/core/errors/src/exception.cpp

```
@@ -439,6 +439,33 @@ namespace hpx {
+    ///////////////////////////////////////////////////////////////////////////
+    std::pair<std::string, error> get_error_info(hpx::exception const& e)
+    {
+        return {e.what(), e.get_error()};
+    }
+
+    std::pair<std::string, error> get_error_info(std::exception_ptr const& e)
+    {
+        auto const error_code = get_error(e);
+        try
+        {
+            std::rethrow_exception(e);
```

Cppcheck (2.21.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)